components/description_list/description_list.test.js

```diff
@@ -0,0 +1,148 @@
+import { mount } from '@vue/test-utils';
+import DtDescriptionList from './description_list.vue';
+
+const baseProps = {
+  items: [{
+    term: 'Customer Intent',
+    description: 'Hello, I\'m looking to return my TV',
+  }, {
+    term: 'Reason',
+    description: 'Refound',
+  }],
+};
+
+let mockProps = {};
+
+describe('DtDescriptionList Tests', () => {
+  let wrapper;
+  let list;
```

Comment:
this should be a `MOCK_LIST`, since it is not sth that is required on the wrapper in initial mount.
Its more like an auxiliary variable you are using to do your stuff.

To be just 'list' I will expect sth like:

```
let wrapper
let list

const updateWrapper = () => {
    wrapper = mount(DtDescriptionList, {
      propsData: { ...baseProps, ...mockProps },
    });

     icon = wrapper.find(...);
  };
```